pg_search/src/postgres/customscan/pdbscan/projections/snippet.rs

```diff
@@ -230,7 +233,7 @@ unsafe fn extract_snippet_positions(
     if let Some(field_arg) = field_arg {
         let attname = (*context)
             .attname_lookup
-            .get(&((*context).rti as _, (*field_arg).varattno as _))
+            .get(&((*context).execution_rti as _, (*field_arg).varattno as _))
```

Comment:
... as to why `extract_snippet_text` is concerned with the "planning_rti" while `extract_snippet_positions` is concerned with the "execution_rti".

It seems to me they both should be using the "planning_rti", no?

Reply:
My bad. You're right. Fixed it.

Reply:
@mdashti now I wonder if this is right?  haha.

`(*context).attname_lookup` is build during planning, with the planning-time `rti` indexes while these `extract_snippet_xxx()` functions are called during execution time (via `create_custom_scan_state()`), which as I (now) understand things, would use execution-time `rti` indexes.

So I wonder if we should move building the `attname_lookup` to `create_custom_scan_state()` too, so that it's happening at execution time?  Quickly skimming the code makes me think we can without any additional fallout.

Reply:
As we discussed, let's do this in a follow-up PR.